Userland/Libraries/LibVideo/PlaybackManager.cpp

```diff
@@ -25,11 +25,11 @@ DecoderErrorOr<NonnullRefPtr<PlaybackManager>> PlaybackManager::from_file(Object
     dbgln_if(PLAYBACK_MANAGER_DEBUG, "Selecting video track number {}", track.identifier());
 
     NonnullOwnPtr<VideoDecoder> decoder = make<VP9::Decoder>();
-    return PlaybackManager::construct(event_handler, demuxer, track, decoder);
+    return make<PlaybackManager>(event_handler, demuxer, track, decoder);
```

Comment:
I'm surprised this compiles, NonnullOwnPtr is not copyable.

I would have written it one of these two ways:
```
    NonnullOwnPtr<VideoDecoder> decoder = make<VP9::Decoder>();
    return make<PlaybackManager>(event_handler, demuxer, track, move(decoder));
```
or
```
    return make<PlaybackManager>(event_handler, demuxer, track, make<VP9::Decoder>());
```

Reply:
huh, good point, not sure what magic is going on there for that to compile :^)

I'll change that, having it inline is totally fine for now.

Reply:
Looks like the reason why it was working was because I passed it as a reference instead of an rvalue reference, silly me :^)

Should pass CI now hopefully, and then it's good to merge!